dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/Intrinsics.java

```diff
@@ -368,6 +375,23 @@ private List<Event> inlineAssert(FunctionCall call) {
         return List.of(assertion, abort);
     }
 
+    private List<Event> integerOverflow(FunctionCall call) {
+        ExpressionFactory expressions = ExpressionFactory.getInstance();
+        final Expression condition = expressions.makeFalse();
+        final Event assertion = EventFactory.newAssert(condition, "integer overflow");
+        final Event abort = EventFactory.newAbortIf(expressions.makeTrue());
+        abort.addTags(Tag.EARLYTERMINATION, Tag.OVERFLOW);
+        return List.of(assertion, abort);
+    }
+
+    private List<Event> nullDereference(FunctionCall call) {
+        final Expression condition = expressions.makeFalse();
+        final Event assertion = EventFactory.newAssert(condition, "invalid dereference");
+        final Event abort = EventFactory.newAbortIf(expressions.makeTrue());
+        abort.addTags(Tag.EARLYTERMINATION, Tag.NULLDEREF);
```

Comment:
It should be `Tag.INVALIDDEREF` or similar (UBSAN checks for more than just null deref).